Decred Wallet/Extensions/UILabel.swift

```diff
@@ -9,12 +9,29 @@
 import UIKit
 
 extension UILabel {
+    
+    private struct AssociatedKeys {
+        static var padding = UIEdgeInsets()
+    }
+
+
     @IBInspectable var xibLocalizedStringKey: String? {
         get { return nil }
         set(key) {
             self.text = NSLocalizedString(key!, comment: "")
             self.setNeedsLayout()
         }
     }
+        
+    public var padding: UIEdgeInsets? {
```

Comment:
What's the relevance of this change?

Reply:
was looking for a solution to the floating label appearing thus:
<img width="325" alt="Screenshot 2019-09-23 at 11 43 40" src="https://user-images.githubusercontent.com/7473976/65419870-87695a80-ddf7-11e9-9ff0-8954d965e05e.png">

this did not solve it. this change is removed in 5419546